eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -280,7 +284,12 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
           blockchain ! WatchLost(self, data.commitments.commitInput.outPoint.txid, nodeParams.minDepthBlocks, BITCOIN_FUNDING_LOST)
           // we make sure that the funding tx has been published
           blockchain ! GetTxWithMeta(funding.commitments.commitInput.outPoint.txid)
-          goto(OFFLINE) using data
+          if (funding.waitingSinceBlock > 1500000) {
+            // we were using timestamps instead of block heights when the channel was created: we reset it to use block heights
+            goto(OFFLINE) using funding.copy(waitingSinceBlock = nodeParams.currentBlockHeight)
```

Comment:
If we don't update the db we'll push back forever:
```suggestion
            goto(OFFLINE) using funding.copy(waitingSinceBlock = nodeParams.currentBlockHeight) storing()
```

Reply:
Good idea!

Reply:
Done in https://github.com/ACINQ/eclair/pull/1692/commits/4019b6693dd52e5537b0c1c7b9db3cde7fa08942